DeveloperLabs/MyEmployees/MyEmployees.sln

```diff
@@ -115,6 +117,26 @@ Global
 		{80DB0989-9307-466B-ABA0-9F82D424CBD8}.Release|x86.ActiveCfg = Release|x86
 		{80DB0989-9307-466B-ABA0-9F82D424CBD8}.Release|x86.Build.0 = Release|x86
 		{80DB0989-9307-466B-ABA0-9F82D424CBD8}.Release|x86.Deploy.0 = Release|x86
+		{8077E806-9F02-48EC-AA21-95B2E8FCC83E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{8077E806-9F02-48EC-AA21-95B2E8FCC83E}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{8077E806-9F02-48EC-AA21-95B2E8FCC83E}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{8077E806-9F02-48EC-AA21-95B2E8FCC83E}.Debug|ARM.Build.0 = Debug|Any CPU
+		{8077E806-9F02-48EC-AA21-95B2E8FCC83E}.Debug|ARM64.ActiveCfg = Debug|Any CPU

```

Comment:
Target ARM for ARM, ARM64 for ARM64, X64 for x64 and x86 for x86 like lines#  105 - 119. Don't target "Any CPU" for specific architectures. I think it's default creation of the VS but you can change that in the sln file.